internal/constants/constants.go

```diff
@@ -0,0 +1,7 @@
+package constants
+
+const (
```

Comment:
This stuff is better to be configurable by config

Ideally, you should not re-build the entire app if you want to change just the DefaultLimit or DefaultPromotedPosts

Reply:
Yeah it would be better in config, and setting defaults also in config as default values.